packages/common/src/index.js

```diff
@@ -256,4 +256,12 @@ const ALERT_SIZES = {
     SLIM: 'slim',
     LARGE: 'large'
 };
-exports.ALERT_SIZES = ALERT_SIZES;
\ No newline at end of file
+exports.ALERT_SIZES = ALERT_SIZES;
+
+const SUPPORT_TYPES = [
```

Comment:
These are in the code as literals a _lot_ of times; seems like it would be a pain to change them if someone decided to change them. At the same time, I'm not sure how nice it would be if (for example) later on someone changes `Maintaining` to `Sustaining`. Then we don't have to change all instances of `'Maintaining'` to `'Sustaining'` but we have the oddity of having many instances of something like a `MAINTAINING` variable that actually contains the string `Sustaining`.

Reply:
I can change the literals to something like SUPPORT_TYPES[0]